dds_web/security/tokens.py

```diff
@@ -17,7 +17,11 @@
 
 # Functions ############################################################################ FUNCTIONS #
 def encrypted_jwt_token(
-    username, sensitive_content, expires_in=datetime.timedelta(hours=48), additional_claims=None
+    username,
+    sensitive_content,
+    expires_in=datetime.timedelta(hours=48),
+    additional_claims=None,
+    fully_authenticated=False,
```

Comment:
I like that solution a lot! It allows to keep the invite/deletion/password reset tokens short, while the regular tokens contain the additional information for a better user experience.